dub.json

```diff
@@ -21,17 +25,33 @@
         },
     },
 
+    "subPackages": [
+        {
+            "name": "configure",
+            "targetType": "executable",
```

Comment:
Ah, nasty. And I presume you don't want to call it directly like `./configure.d` to keep it more in style with configure-make pattern?